### PR TITLE
Add basic docker env

### PR DESCRIPTION
**Pull request by dayllanmaza**
Bug: https://phabricator.wikimedia.org/T277668

### CODE REVIEW

File: docker/run.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+docker run -p ${DOCKER_OCR_PORT:-8000}:8000 --mount type=bind,source="$(pwd)",target=/wikimedia-ocr wikimedia-ocr:latest
```
Comment (samwilson) [inline docker/run.sh]: This could have `source="$(cd "$(dirname $0)/.."; pwd)"` to guard against when it's run from a different directory.

**Review by samwilson — COMMENTED** [review]
I haven't fully got this working yet, but just noticed this one thing.

File: docker/run.sh (hunk 1 above)
Comment (dayllanmaza) [reply docker/run.sh]: Thanks

**Review by MusikAnimal — COMMENTED** [review]
Works in my testing! However I'm not sure how to _stop_ the container. `./docker/run.sh` spins up the symfony server and I see the output, but the usual Ctrl+C doesn't kill it. I actually had to run `kill -9` on the process.

**Comment by dayllanmaza** [comment]
> Works in my testing! However I'm not sure how to _stop_ the container. `./docker/run.sh` spins up the symfony server and I see the output, but the usual Ctrl+C doesn't kill it. I actually had to run `kill -9` on the process.

It should handle Ctrl+C now. Alternatively, you can also run `docker ps` and `docker stop|kill <id or name>` to stop the container

**Pull request merged**